### PR TITLE
feature - detail ui

### PR DESCRIPTION
**Pull request by at-tungnguyen**
#### What's this PR do? Please give a technical summary `(required)`
- [x] finish detail UI
#### Any references (stackoverflow …) `(optional)`

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 1]
```diff
@@ -32,44 +38,70 @@ class DetailViewController: ViewController {
...
+
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return 20
+    }
+
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        let footerView = UIView()
+        footerView.backgroundColor = UIColor.clear
+        return footerView
     }
```
Comment (at-linhvo) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: ```detailTableView.tableFooterView = UIView()```
``` detailTableView.sectionFooterHeight = 20```

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 2]
```diff
@@ -32,44 +38,70 @@ class DetailViewController: ViewController {
...
-            cell.textLabel?.text = "Description"
+        } else {
+            guard let rowNumber = RowNumber(rawValue: indexPath.row) else {
+                return UITableViewCell()
+            }
+            let cell = tableView.dequeue(InfomationDetailTableViewCell.self)
+            switch rowNumber {
+            case .name:
+                cell.titleLabel.text = "Name"
+                cell.symbolImageView.image = #imageLiteral(resourceName: "ic_tag")
```
Comment (at-linhvo) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: define enum

**Comment by at-tungnguyen** [comment]
Please check this pull request. Thank you

**Pull request merged**